src/pages/GapsPage.tsx

```diff
@@ -79,6 +80,14 @@ const GapsPage = () => {
       .finally(() => setRoutesIsLoading(false))
   }, [operatorId, lineNumber, timestamp, setSearch])
 
+  const gapsPrecentage = useMemo(() => {
```

Comment:
useMemo is not necessary for primitive values such as numbers
it is a good idea to make a function that takes both arrays and output the value